packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -66,6 +66,8 @@ export interface MergedTransaction {
   l1ToL2MsgData?: L1ToL2MessageData
   l2ToL1MsgData?: L2ToL1MessageData
   depositStatus?: DepositStatus
+  chainId?: number // tx's chain id - L2 / L3
+  parentChainId?: number // tx's parent chain - L1 / L2
```

Comment:
Technically the parent chain could be an Orbit chain too, e.g. if it's an L3 <> L4 tx. We might want to keep the comments scalable.

I'd say maybe: `// tx's parent chain` is enough